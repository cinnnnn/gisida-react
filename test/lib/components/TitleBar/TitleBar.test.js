import React from 'react';
import TitleBar from '../../../../src/lib/components/TitleBar/TitleBar.js'
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';

const initialState = {
	APP: { loaded: true } 
}

const mockStore = configureStore()
const store = mockStore(initialState);

const componentWrapper = mount(
	<TitleBar 
		store={store}
	/>
);

describe('TitleBar', () => {

	it('TitleBar component renders correctly', () => {
		const json = toJson(componentWrapper)
		expect(json).toMatchSnapshot();
	})
});