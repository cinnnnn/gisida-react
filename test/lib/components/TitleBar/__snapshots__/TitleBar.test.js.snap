// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleBar TitleBar component renders correctly 1`] = `<div />`;
